app/src/main/java/com/amaze/filemanager/filesystem/FolderState.kt

```diff
@@ -27,6 +27,7 @@ const val WRITABLE_OR_ON_SDCARD = 1
 
 // For Android 5
 const val CAN_CREATE_FILES = 2
+const val WRITABLE_ON_REMOTE = 3
```

Comment:
Is introducing new const okay? I see many operations like delete Files, compress, extract, that are calling `checkFolder` displays `Operation not supported` in their else cases.

Reply:
Flag was introduced so https://github.com/TranceLove/AmazeFileManager/blob/ea5aca20d0151b421a687c21944615cffb475d5b/app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/PrepareCopyTask.java#L331 would pass.

MainActivityHelper.deleteFiles() automatically goes to DeleteTask right away if the file(s) is/are SMB. See https://github.com/TranceLove/AmazeFileManager/blob/ea5aca20d0151b421a687c21944615cffb475d5b/app/src/main/java/com/amaze/filemanager/utils/MainActivityHelper.java#L630-L633

But for extract/compress files, saw the problem - can work more to accommodate the flag.